fluster/codec.py

```diff
@@ -51,7 +51,8 @@ class OutputFormat(Enum):
     GBRP12LE = "gbrp12le"
     GBRP14LE = "gbrp14le"
     GRAY = "gray"
-    GRAY10LE = "gray10le"
+    GRAY10LE16 = "gray10le16"
+    GRAY10LE32 = "gray10le32"
```

Comment:
I don't think introducing GRAY10LE32 make any sense here. Please drop.

fyi, this is 3 pixels per 32bit packing, with 2 bit padding, we should nto do conformance on that.

Reply:
I also think we should keep gray10le to match with ffmpeg name, and is 16bit per pixels.